### PR TITLE
Change diagnostic page icon to a stethoscope

### PR DESCRIPTION
**Pull request by TTitcombe**
**Description of work.**
The ISIS SANS diagnostic page icon has been changed to a stethoscope, to distinguish it from the Help page icon.
The settings page icon changed from a dark cog to an outline of a cog. This brings it more inline with the surrounding icons, which are predominantly white.

**To test:**
1. Interfaces -> SANS -> ISIS SANS
2. Look at the tabs on the left hand side of the main gui. There should be a play button, a cog, a circle with a dot, a circle with a plus, and a stethoscope.
3. Do the cog or stethoscope look out of place or ambiguous?

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py [hunk 1]
```diff
@@ -289,7 +289,7 @@ def setup_layout(self):
         runs_icon = icons.get_icon("mdi.play-circle-outline")
         _ = QListWidgetItem(runs_icon, "Runs", self.tab_choice_list)  # noqa
 
-        settings_icon = icons.get_icon("mdi.settings")
+        settings_icon = icons.get_icon("mdi.settings-outline")
```
Comment (Pasarus) [inline scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]: To stay in line with the other GUIs and general icon usage could we stick to "mdi.settings"
```suggestion
        settings_icon = icons.get_icon("mdi.settings")
```

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py [hunk 2]
```diff
@@ -289,7 +289,7 @@ def setup_layout(self):
         runs_icon = icons.get_icon("mdi.play-circle-outline")
```
Comment (Pasarus) [inline scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]: Much like with the settings icon can we keep this inline with the other parts of workbench Icons.
```suggestion
        runs_icon = icons.get_icon("mdi.play")
```

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py [hunk 3]
```diff
@@ -298,7 +298,7 @@ def setup_layout(self):
         add_runs_page_icon = icons.get_icon("mdi.plus-circle-outline")
```
Comment (Pasarus) [inline scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]:
```suggestion
        add_runs_page_icon = icons.get_icon("mdi.sigma")
```

**Review by Pasarus — CHANGES_REQUESTED** [review]
Looking good, but with the change of stethoscope to abandon the circle around the icon I think we should follow suit with all icons others than the beam centre as I think that Icon is ideal.

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py (hunk 1 above)
Comment (TTitcombe) [reply scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]: That's been changed back now

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py (hunk 2 above)
Comment (TTitcombe) [reply scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]: Good suggestion

File: scripts/Interface/ui/sans_isis/sans_data_processor_gui.py (hunk 3 above)
Comment (TTitcombe) [reply scripts/Interface/ui/sans_isis/sans_data_processor_gui.py]: I've removed the circle, but kept plus as sigma is used elsewhere in Mantid to denote processing

**Review by Pasarus — APPROVED** [review]

**Pull request merged**